CMakeLists.txt

```diff
@@ -40,7 +40,11 @@ file(GLOB UOSM_CORE_SOURCES
         UOSM-Core/Utils/*.cpp
 )
 
-add_executable(UOSM_Telemetry_Pico ${SOURCES} ${UOSM_CORE_SOURCES})
+add_executable(UOSM_Telemetry_Pico ${SOURCES} ${UOSM_CORE_SOURCES}
+        Core/Tasks/CurrentVoltageTask.h
+        Core/Tasks/CurrentVoltageTask.cpp
+        Core/Tasks/Accelerometer.c
+        Core/Tasks/accelerometer.h)
```

Comment:
The .h files are included using the include directories command at line 20.
Shouldn't be needed